elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -129,31 +130,39 @@ private Elide(AuditLogger auditLogger,
      * Elide Builder for constructing an Elide instance.
      */
     public static class Builder {
-        private final AuditLogger auditLogger;
         private final DataStore dataStore;
-        private EntityDictionary entityDictionary;
+        private AuditLogger auditLogger;
         private JsonApiMapper jsonApiMapper;
-        private Function<RequestScope, PermissionExecutor> permissionExecutorFunction;
+        private EntityDictionary entityDictionary = new EntityDictionary(new HashMap<>());
+        private Function<RequestScope, PermissionExecutor> permissionExecutorFunction = ActivePermissionExecutor::new;
 
         /**
-         * Constructor.
+         * A new builder used to generate Elide instances. Instantiates an {@link EntityDictionary} without
+         * providing a mapping of security checks.
          *
-         * @param auditLogger
-         * @param dataStore
+         * @param auditLogger the logger to use for audit annotations
+         * @param dataStore the datastore used to communicate with the persistence layer
+         * @deprecated 2.3 use {@link #Builder(DataStore)}
          */
         public Builder(AuditLogger auditLogger, DataStore dataStore) {
             this.auditLogger = auditLogger;
             this.dataStore = dataStore;
-            this.entityDictionary = new EntityDictionary(new HashMap<>());
             this.jsonApiMapper = new JsonApiMapper(entityDictionary);
-            this.permissionExecutorFunction = null;
+        }
+
+        /**
+         * A new builder used to generate Elide instances. Instantiates an {@link EntityDictionary} without
+         * providing a mapping of security checks and uses the provided {@link Slf4jLogger} for audit.
+         *
+         * @param dataStore the datastore used to communicate with the persistence layer
+         */
+        public Builder(DataStore dataStore) {
+            this.dataStore = dataStore;
+            this.auditLogger = new Slf4jLogger();
```

Comment:
We should probably add an `Builder auditLogger(AuditLogger auditLogger);` method to the builder, otherwise auditLogger cannot be overwritten given the previous builder is deprecated